lib/config.js

```diff
@@ -17,6 +18,12 @@ function getNodeVersion () {
 
 async function getGitRev () {
   let cwd = path.resolve(__dirname, "..", "..");
+
+  // Check if we're in the right folder, relative to package.json
+  if (!await fs.exists(path.resolve(cwd, 'package.json'))) {
```

Comment:
is there a way to know based on some external criteria, rather than just guessing and trying one and then another?

Reply:
There's a package called `find-root`. Would that be good?